.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Documentation
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+version: 2
+updates:
+-   package-ecosystem: github-actions
```

Comment:
typically this will trigger a bunch of automated PR after merge to update all the actions